Registering prefix contact details (#1130)
open-contracting/standard: docs/guidance/build.md

To publish OCDS data, you need to register an ocid prefix.

> **Action**: Email <data@open-contracting.org> to request an OCID prefix. Provide the name of your organization and the email address of a contact person, who is an employee of the publisher.

Reviewer: Is "who is an employee of the publisher" to avoid a consultant putting their name forward? In that case, we should align "name of your organization", e.g.:
```suggestion
**Action**: Email <data@open-contracting.org> to request an OCID prefix. Provide the name of the publishing organization and the email address of a contact person at this organization.
```
